crypto/objects/obj_dat.pl

```diff
@@ -188,74 +174,50 @@ sub expand_obj
 
 EOF
 
-printf OUT "#define NUM_NID %d\n",$n;
-printf OUT "#define NUM_SN %d\n",$#sn+1;
-printf OUT "#define NUM_LN %d\n",$#ln+1;
-printf OUT "#define NUM_OBJ %d\n\n",$#ob+1;
-
-printf OUT "static const unsigned char lvalues[%d]={\n",$lvalues+1;
+printf OUT "static const unsigned char o[%d] = {\n", $lvalues + 1;
```

Comment:
May I suggest a slightly different name?  I understand you want it to be short, but what about `_o` or `o_`?

The reason I point this out is that if you have a look in `crypto/objects/obj_dat.c`, you'll find lines like this:

    ASN1_OBJECT *o;

    ASN1_OBJECT o;

I'm pretty sure you'll find builds in Travis where the compiler will complain *harshly* about symbol shadowing.